README.md

```diff
@@ -292,31 +163,31 @@ all:
     pemserver:
       hosts:
         pemserver1:
-          ansible_host: xxx.xxx.xxx.xxx
-          private_ip: xxx.xxx.xxx.xxx
+          ansible_host: 110.0.0.4
+          private_ip: 10.0.0.4
     primary:
       hosts:
         primary1:
-          ansible_host: xxx.xxx.xxx.xxx
-          private_ip: xxx.xxx.xxx.xxx
+          ansible_host: 110.0.0.1
+          private_ip: 10.0.0.1
           pem_agent: true
-          pem_server_private_ip: xxx.xxx.xxx.xxx
+          pem_server_private_ip: 10.0.0.4
     standby:
       hosts:
         standby1:
-          ansible_host: xxx.xxx.xxx.xxx
-          private_ip: xxx.xxx.xxx.xxx
-          upstream_node_private_ip: xxx.xxx.xxx.xxx
+          ansible_host: 110.0.0.2
+          private_ip: 10.0.0.2
+          upstream_node_private_ip: 10.0.0.1
           replication_type: synchronous
           pem_agent: true
-          pem_server_private_ip: xxx.xxx.xxx.xxx
+          pem_server_private_ip: 10.0.0.4
         standby2:
-          ansible_host: xxx.xxx.xxx.xxx
-          private_ip: xxx.xxx.xxx.xxx
-          upstream_node_private_ip: xxx.xxx.xxx.xxx
+          ansible_host: 110.0.0.3
+          private_ip: 10.0.0.3
+          upstream_node_private_ip: 10.0.0.1
           replication_type: asynchronous
           pem_agent: true
-          pem_server_private_ip: xxx.xxx.xxx.xxx
+          pem_server_private_ip: 10.0.0.4
```

Comment:
Do we want to keep IP addresses?

Reply:
Yes. I remember someone complaining about not getting real examples in terms of IP address configuration